src/lib/stores/index.ts

```
@@ -154,43 +155,77 @@ type Settings = {
-	STTEngine?: string;
-	TTSEngine?: string;
-	speaker?: string;
-	model?: string;
-	nonLocalVoices?: boolean;
+export type AudioSettings = {
+	stt?: {
+		engine?: string;
+	};
+	tts?: {
+		engine?: string;
+		engineConfig?: Record<string, any>;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)